Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/revolvers.yml

```diff
@@ -5,9 +5,9 @@
   description: Argenti Type 20 revolver. Manufactured by Silver Industries. While the design with expanded cylinder is quite ancient, the right gunslinger will know how to utilise it well. Uses .20 rifle ammo.
   components:
   - type: Sprite
-    sprite: _NF/Objects/Weapons/Argenti-Revolver.rsi
+    sprite: _NF/Objects/Weapons/Guns/Revolvers/Argenti-Revolver.rsi
   - type: Clothing
-    sprite: _NF/Objects/Weapons/Argenti-Revolver.rsi
+    sprite: _NF/Objects/WeaponsGuns/Revolvers/Argenti-Revolver.rsi
```

Comment:
this looks like a typo here, should probably fix

Reply:
Fix it